webpack.config.js

```diff
@@ -0,0 +1,26 @@
+var webpack = require('webpack');
+var path = require('path');
+
+var SRC_DIR = path.join(__dirname, '/client/src');
+var DIST_DIR = path.join(__dirname, '/client/dist');
+
+module.exports = {
+  entry: `${SRC_DIR}/index.jsx`,
+  target: 'node',
+  output: {
+    filename: 'main.js',
+    path: DIST_DIR
+  },
+  module: {
+    loaders: [
+      {
+        test: /\.m?js$/,
+        include: SRC_DIR,
+        loader: 'babel-loader',
+        query: {
+          presets: ['@babel/preset-env', '@babel/preset-react']
+        }
+      }
+    ]
+  }
```

Comment:
Is this webpack config working for you? I think you're using the same version I am, and I had to change some of these `module` properties before I could get it to work. Webpack config is tricky (for me at least). Check out my latest version if you run into trouble with this config.

Reply:
I had the same issues in my code, and resolved them by following Justin's config example